src/mscorlib/src/System/Runtime/Intrinsics/X86/Avx.PlatformNotSupported.cs

```diff
@@ -438,58 +438,58 @@ public static class Avx
         /// __m256i _mm256_insertf128_si256 (__m256i a, __m128i b, int imm8)
         ///   VINSERTF128 ymm, ymm, xmm/m128, imm8
         /// </summary>
-        public static Vector256<T> Insert<T>(Vector256<T> value, Vector128<T> data, byte index) where T : struct { throw new PlatformNotSupportedException(); }
+        public static Vector256<T> InsertVector128<T>(Vector256<T> value, Vector128<T> data, byte index) where T : struct { throw new PlatformNotSupportedException(); }
```

Comment:
I don't see the corresponding change in the `Avx.cs` file (the one that isn't PlatformNotSupported).